syserol/classify.py

```diff
@@ -52,8 +50,10 @@ def two_way_classifications():
 
 def ClassifyHelper(X, Y):
     """ Function with common Logistic regression methods. """
-    regr = LogisticRegressionCV(n_jobs=-1, cv=40, max_iter=1000).fit(X, Y)
```

Comment:
I don't understand what you're trying to do here.

If you're not understanding the process of cross-validation and regularization weight selection, we should make sure you understand those concepts before doing anything more. It's important that you understand the reasoning here because this looks like cargo cult modeling—it might run but is completely wrong.

Reply:
Ah sorry. In the Alter paper, they said they ran the modeling 100 times and chose the result with median accuracy as the representative model, so I was attempting to recreate that - I was assuming that glmnet was doing the cross validation internally. However, I do realize that I am confused about the cross validation.

Could you tell me if this would be correct? Do a stratified Kfold to create multiple groups. Within each fold, fit logitnet with the training group (Which does it's own cv internally) and then predict the test group and find accuracy. Repeat for each fold, and take the average of the accuracies to be the final number, and the final Y_pred as the concatenation of those from each fold?

Reply:
Also, is this PR necessary anymore? You seem to have already done it yourself.